CONTRIBUTING.md

```diff
@@ -87,9 +87,19 @@ Your contribution needs to meet the following standards:
   Co-authored-by: <B full name> <B email>
   ```
 
-- Usage of `unsafe` should be documented with a justification (over its safe
-  alternative where one exists) and an explanation on why it cannot produce
-  undefined behavior. E.g.
+- If something can be done in safe Rust, we will not use
```

Comment:
O, I like that. I did keep the part about listing invariants though, because I've seen that not happen a few times and I think its important for a good safety comment (and clippy docs sadly don't really include anything on best practices for writing these comments 😞)